Examples/Profiles/Profile-example.fsh

```diff
@@ -1,4 +1,39 @@
 // @Name: Profile Example
+// @Description: Example of a profile of the Patient resource. This example includes a few of the most commonly used constraints and documentation features of FHIR profiles.
 
-// Nothing yet! 
-// Help wanted -- your example can go here. 
\ No newline at end of file
+Profile:     ExamplePatientProfile
+Id:          example-patient-profile
+Parent:      Patient
+Title:       "Example Patient Profile"
```

Comment:
Since below, you are very explicit about where `short` and `definition` are displayed, would it make sense to also put a comment on Title and Description keywords explaining where they go?

Reply:
👍, done